CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Update `ProposalActions` module component to improve rendering of decoded parameters and support and "edit" mode
   through the `editMode` property
 
+### Fixed
```

Comment:
The new entry should be under the `unreleased` section, seeing this now sorry 😬

Reply:
My bad, I missed that, as I think it changed between when I started working on it and now. Updated